docs/docs/migration-guide.mdx

```diff
@@ -437,6 +437,41 @@ response_selector_output.get("default")
                         .get("intent_response_key")
 ```
 
+### Core Featurization
+
+* `LabelTokenizerSingleStateFeaturizer` is deprecated and will be removed in the future.
+  Please use `SingleStateFeaturizer` with appropriate NLU pipeline.
+  To replicate `LabelTokenizerSingleStateFeaturizer` functionality,
+  add a `Tokenizer` with the option `intent_tokenization_flag: True` and `CountVectorsFeaturizer`
+  to the NLU pipeline. Please update your configuration file.
+
+  For example:
+  ```yaml
```

Comment:
```suggestion
  ```yaml-rasa {3-5}
```

Reply:
Whenever you provide yaml for training data, endpoints, configs etc, `yaml-rasa` will make sure it is valid yaml and passes any relevant specs. the numbers in the curly braces will highlight specific lines